### DISCUSSION TITLE
Full Height QTabPanel containing Full Height Scroll Area

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#12920] [General - Components / Directives / etc] [answered]
How can I place a Scroll Area within my QTabPanel and have it fill the available height? In my example on CodePen, i have 3 tabs all which contain various content that i want to fit to the height of the available space and scroll within the Tab when necessary.

https://codepen.io/jokermartini/pen/PoEWWge

There is a codepen that gets close to working for a slightly similar issue but doesn't solve my problems. https://codepen.io/pdanpdan/pen/MWbJzPd?editors=1100

In the last tab 'Notes' i have a vertical splitter which I want to fit within the available height and make a scroll area within the TabPanel. However no matter what i try to do i can't get this to work, it extends off the page.

![image](https://user-images.githubusercontent.com/1416749/159937889-2da15828-715c-492f-8622-89681d633d40.png)

Again you'll notice in the second tab it doesn't extend the available height which i want it to.

![image](https://user-images.githubusercontent.com/1416749/159938130-0374002f-910c-46d4-8606-32ddb8798464.png)

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/PoEWQBN?editors=1010

You want to use a `column no-wrap` on parent to allow for dynamic fit of the children.
Take care - `QPage` has a `min-height` by default and no `height` so in order to make column/col work you need to set a fixed height.
And in the panels you need to take care to make them scrollable.
